tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
```

Comment:
Did you test how this may affect other imports at runtime?

Reply:
No, but it won't because we use babel everywhere else.

    "esModuleInterop": true

is just

    "allowSyntheticDefaultImports": true

plus tsc emitting the import shim that babel emits by default.

Reply:
Cool cool, sounds good 👍